### PR TITLE
Expose config values from dpos module - Closes #7213

### LINKED ISSUE #7213: Expose config values from dpos module
### Description

Expose numberOfActiveDelegate and numberOfStandbyDelegate from DPoS module and also check for other values to expose.

Expose
`dpos_getConstants` endpoint with response of 
```
{
	factorSelfVotes: number;
	maxLengthName: number;
	maxNumberSentVotes: number;
	maxNumberPendingUnlocks: number;
	failSafeMissedBlocks: number;
	failSafeInactiveWindow: number;
	punishmentWindow: number;
	roundLength: number;
	bftThreshold: number;
	minWeightStandby: bigint;
	numberActiveDelegates: number;
	numberStandbyDelegates: number;
	tokenIDDPoS: TokenIDDPoS;
}
```

### Which version(s) does this affect? (Environment, OS, etc...)

`v6.0.0-alpha.0`

### PR DESCRIPTION
**Pull request by has5aan**
### What was the problem?

This PR resolves #7213

### How was it solved?

Under `framework/src/modules/dpos_v2` modified:

- `type.ts` - defines `ModuleConfigJSON` type.
- `endpoint.ts` - defines `getConstants` endpoint and `init` function to initialise, `ModuleConfig`.
- `module.ts` - calls `DPoSEndPoint.init` providing the module config and configures `getConstants` end-point.
- `utils.ts` - defines `getModuleConfig`, converts `ModuleConfigJSON` to `ModuleConfig`.

Under `framework/test/unit/modules/dpos_v2` modified / created:
- `endpoint.ts` - tests `getConstants` endpoint.
- `utils.ts` - tests `getModuleConfig` utility function.

### How was it tested?

Passing tests.

Testing the response of the getConstants by configuring the modified module with a test node. An approach would be to use the `APIClient`:

```javascript
require('@liskhq/lisk-api-client').createIPCClient('~/.lisk/dpos-mainchain').then(client => {
    client.invoke('dpos_getConstants').then(response => {
        console.log(response)
    })
})
```

### CODE REVIEW

File: framework/src/modules/dpos_v2/schemas.ts [hunk 1]
```diff
@@ -655,3 +655,66 @@ export const getAllDelegatesResponseSchema = {
 		},
 	},
 };
+
+// export const getConstantsResponseSchema = {
```
Comment (shuse2) [inline framework/src/modules/dpos_v2/schemas.ts:659]: why comment out?

File: framework/src/modules/dpos_v2/types.ts [hunk 2]
```diff
@@ -32,6 +32,22 @@ export interface ModuleConfig {
 	tokenIDDPoS: TokenIDDPoS;
 }
 
+export interface ModuleConfigJSON {
```
Comment (shuse2) [inline framework/src/modules/dpos_v2/types.ts:35]: you could use
```
export type ModuleConfigJSON = JSONObject<ModuleConfig>;
```

**Review by shuse2 — COMMENTED** [review]
Generally LGTM, just minor point and the jenkins is failing

File: framework/src/modules/dpos_v2/types.ts (hunk 2 above)
Comment (has5aan) [reply framework/src/modules/dpos_v2/types.ts]: Have changed the code based on the suggestions. Thanks.

**Review by ishantiw — APPROVED** [review]
Tested the endpoint and looks good 👏

**Pull request merged**